lib/spack/spack/package.py

```diff
@@ -380,6 +380,22 @@ def __init__(self, spec):
         if not hasattr(self, 'list_depth'):
             self.list_depth = 1
 
+        # Set default licensing information
+        if not hasattr(self, 'license_required'):
+            self.license_required = False
+
+        if not hasattr(self, 'license_comment'):
+            self.license_comment = '#'
+
+        if not hasattr(self, 'license_files'):
+            self.license_files = []
+
+        if not hasattr(self, 'license_vars'):
+            self.license_vars = []
+
+        if not hasattr(self, 'license_url'):
+            self.license_url = None
```

Comment:
@adamjstewart I wonder in which way (the `spec`-`package` mechanics are still somewhat obscure to me...). I guess I'll need to go through the code with a debugger to get a better grasp :smile: